tests/cmd/package/out/Proto.toml

```diff
@@ -2,3 +2,5 @@
 type = "lib"
 name = "lib"
 version = "0.0.1"
+
+[dependencies]
```

Comment:
it'd be nice to have some tests here which cover lockfiles

Reply:
That's fair, but unfortunately that'd require registry interactions which we're currently unable to mock. I think #26 and #30 partially cover that, but I'll add a note about this specifically.